addons/website/views/website_templates.xml

```diff
@@ -407,12 +407,6 @@
                                 <a href="/website/social/instagram" class="s_social_media_instagram" target="_blank">
                                     <i class="fa fa-instagram m-1"/>
                                 </a>
-                                <a href="/website/social/youtube" class="s_social_media_youtube d-none" target="_blank">
```

Comment:
Commit message:
- Typo: unecessary
- Clearer: "remove hidden social media links" or something like that. So that we immediately know what this commit is about